platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -2349,7 +2349,7 @@
 		DA25D5BC1CCD9EDE00607828 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				INFOPLIST_FILE = framework/Settings.bundle/Root.plist;
+				INFOPLIST_FILE = "framework/Info-static.plist";
```

Comment:
It would. Same goes for the Bundle.bundle which the static target also depends on. (I copied this from the bundle target)
Is the Settings.bundle only meant to be used in iosapp?